modules/distribution/email-admin-config.xml

```diff
@@ -0,0 +1,65 @@
+<!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<!-- 
+     Contains the body of the mail that to be sent upon the tenant configuration request by the 
+     admin of the tenant. This includes the password reset link, for the forgotten passwords.
+  -->
+
+<configuration>
+    <targetEpr>https://localhost:9443/carbon/admin-mgt/validator_ajaxprocessor.jsp</targetEpr>
+    <subject>WSO2 Carbon - Password Reset</subject>
+    <body>
+        Hi {first-name},
+
+        A password reset request has recently been sent by you (or someone else).
+
+        User Name: {user-name}
+
+        Please click the following link to reset your password.
+    </body>
+    <footer>
+        Best Regards,
+        WSO2 Carbon Team
+        http://www.wso2.com
+    </footer>
+    <redirectPath>../admin-mgt/update_verifier_redirector_ajaxprocessor.jsp</redirectPath>
+</configuration>
+<configuration type="askPassword">
+<targetEpr></targetEpr>
+<subject>WSO2 MDM - Change password of the new user account</subject>
+<body>
+    Hi {first-name},
+
+
+    Please change your password for the newly created account: {user-name}. Please click the link
+    below to create the password.
+    http://localhost:9443/InfoRecoverySample/infoRecover/verify?confirmation={confirmation-code}&
+    username={user-name}
+
+
```

Comment:
This is directly copied form https://github.com/wso2/product-emm/blob/master/modules/distribution/email-admin-config.xml